packages/web-components/src/components/va-process-list/va-process-list.tsx

```diff
@@ -11,15 +11,30 @@ import { Component, h } from '@stencil/core';
  */
 @Component({
   tag: 'va-process-list',
-  styleUrl: 'va-process-list.css',
+  styleUrl: 'va-process-list.scss',
   shadow: true,
 })
 export class VaProcessList {
+  /**
+   * Whether or not the component will use USWDS v3 styling.
+   */
+  @Prop() uswds?: boolean = false
+
   render() {
-    return (
-      <ol role="list">
-        <slot></slot>
-      </ol>
-    );
+    const { uswds } = this;
+    
+    if (uswds) {
```

Comment:
Split based on `uswds` by default, but both branches return the same. Should this be removed or left in for potential future changes?